.github/workflows/deb_release.yml

```diff
@@ -0,0 +1,73 @@
+name: Debian packages release
+
+on:
+  workflow_dispatch:
+    inputs:
+      config:
+        description: JSON of options
+        type: string
+        default: '{}'
+        required: true
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Save the config options
+        run: |
+          echo '${{ inputs.config }}' | jq > config.json
+      - name: Archive the release configuration
+        uses: actions/upload-artifact@v3
+        with:
+          name: config.json
+          path: config.json
+      - name: Install dependencies
+        run: |
+          sudo apt update -qq
+          sudo apt install -qq -y bumpversion devscripts python3-launchpadlib
+          git clone https://git.launchpad.net/~checkbox-dev/checkbox/+git/support
+      - name: Checkout checkbox monorepo
+        uses: actions/checkout@v3
+        with:
+          repository: canonical/checkbox
+          fetch-depth: 0
+          path: src/checkbox
+      - name: Check for new commits
+        run: |
+          support/release/git/release_deb_monorepo.py clone --config config.json
+      - name: Bump versions
+        run: |
+          git config --global user.email "sylvain.pineau@canonical.com"
+          git config --global user.name "Sylvain Pineau"
+          support/release/git/release_deb_monorepo.py bump --part major
+      - name: Create changelog
+        run: |
+          support/release/git/release_deb_monorepo.py changelog
+      - name: Archive changelog
+        uses: actions/upload-artifact@v3
+        with:
+          name: Changelog
+          path: changelog
+      - name: Push release tags
+        run: |
+          support/release/git/release_deb_monorepo.py push
+      - name: Update the PPA recipe and kick-off the builds
+        env:
+          LP_CREDS: credentials
+        run: |
+          echo '${{ secrets.LP_CREDS }}' > credentials
+          support/release/git/release_deb_monorepo.py build
+          rm -f credentials
+      - name: Open a new release for development
+        if: fromJson(inputs.config).mode == 'stable'
+        run: |
+          support/release/git/release_deb_monorepo.py open
+      - name: Create pull request
+        if: fromJson(inputs.config).mode == 'stable'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DEBEMAIL: sylvain.pineau@canonical.com
+          DEBFULLNAME: Sylvain Pineau
+        working-directory: src/checkbox
+        run: |
+          gh pr create -H release --title 'Open a new release for development' --body 'Created by Github action'
```

Comment:
I can live with this as a temporary measure if bump-version really doesn't have a useful behaviour with `--no-commit`, but it's also a tiny bit weird to do this, so it will be a temporary measure for sure.